python-lib/ontology_tagger.py

```diff
@@ -70,7 +81,7 @@ def _remove_incomplete_rows(self) -> None:
             )
 
     def _get_patterns(
-        self, list_of_keywords: List[AnyStr], list_of_tags: List[AnyStr]
+        self, list_of_keywords: List[AnyStr], list_of_tags: List[AnyStr], language
```

Comment:
Nitpick: add type annotation for language to remain consistent with the other arguments